modules/codegen/src/main/scala/com/twilio/guardrail/SwaggerUtil.scala

```diff
@@ -350,14 +350,14 @@ object SwaggerUtil {
     } else Option.empty[L#Type].pure[F]
   }
 
-  def propMetaWithName[L <: LA, F[_]](tpe: L#Type, property: Tracker[Schema[_]])(
+  def propMetaWithName[L <: LA, F[_]](tpe: L#Type, property: Tracker[Schema[_]], arrayTypeLifter: (L#Type, Option[L#Type]) => F[L#Type])(
```

Comment:
I don't love `arrayTypeLifter`, but one of my goals here was to use the same type resolution code for protocol objects and parameters.

Reply:
Hrm, it appears that this ends up using the abstract container type for server routes, which almost certainly will not work.

Reply:
Actually, I guess that was always happening.  For Java stdlib it's not a problem since it's always a concrete type, and I guess for the Scala stuff the routes don't have types listed?  But for Java Vavr I'm using `Seq` now, which will not work.  I guess I can go back to `Vector`.  Hrm.